apps/api/pages/api/event-types/_get.ts

```diff
@@ -56,6 +56,7 @@ async function getHandler(req: NextApiRequest) {
       owner: { select: { username: true, id: true } },
     },
   });
+  if(data.length === 0)  new HttpError({ statusCode: 404, message: "No event types were found" });
```

Comment:
@Nikeshkumar-tk Can you fix the spacing here? 
- Add a space between `if` and `(`.
- Remove 1 of the 2 spaces before `new`

Reply:
```suggestion
  if (data.length === 0) new HttpError({ statusCode: 404, message: "No event types were found" });
```